roles/contiv_network/templates/netplugin.j2

```diff
@@ -1 +1,6 @@
-NETPLUGIN_ARGS=""
+{# If an interface is specified for netplugin, use it #}
+{% if netplugin_if is defined %}
+NETPLUGIN_ARGS='-docker-plugin -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}}'
+{% else %}
+NETPLUGIN_ARGS='-docker-plugin -vtep-ip {{node_addr}}'
```

Comment:
As I understand, the fix in netplugin repo will handle it in a way that it will error out gracefully. Will let @shaleman comment on this. Again, will handle it after raising an issue if required. Have other priorities to work on currently.